src/index.html

```diff
@@ -1,13 +1,381 @@
+      rel="shortcut icon"
+      type="image/x-icon"
+    >
+    <script
+      defer
+      src="scripts/main.js"
+    ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page_body">
+    <header class="header">
+      <div class="header_top">
+        <a
+          class="menu_logo"
+          href="#"
+        >
+          <img
+            alt="Bike Logo"
+            class="logo"
+            src="images/logo.png"
+          >
+        </a>
+        <div class="header_icons">
+          <a
+            class="icon icon_phone"
+            href="tel:+1 234 5555-55-55"
+          ></a>
+          <a
+            class="icon icon_menu"
+            href="#menu"
+          ></a>
+        </div>
+      </div>
+      <div class="header_bottom">
+        <h1 class="header_title">Take the Streets</h1>
+      </div>
+    </header>
+
+    <aside
+      class="page_menu menu"
+      id="menu"
+    >
+      <div class="header_top menu_top">
+        <a
+          class="menu_logo"
+          href="#"
+        >
+          <img
+            alt="Bike Logo"
+            class="logo"
+            src="images/logo__menu.png"
+          >
+        </a>
+        <a
+          class="icon icon_close"
+          href="#"
+        ></a>
+      </div>
+
+      <div class="menu_content">
+        <nav class="nav">
+          <ul class="nav_list ">
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#"
+              >Home</a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#about_us"
+              >About us</a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#compare_bikes"
+              >Compare Bikes</a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#details"
+              >Details</a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link"
+                href="#contacts"
+              >Contacts</a>
+            </li>
+          </ul>
+          <ul class="nav_list nav_list_bottom">
+            <li class="nav_item">
+              <a
+                class="nav_link nav_link_tel"
+                href="tel:+1 234 5555-55-55"
+              >+1 234 5555-55-55</a>
+            </li>
+            <li class="nav_item">
+              <a
+                class="nav_link nav_link_book"
+                href="#test_ride"
+              >Book a test ride</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </aside>
+
+    <main class="main">
+      <section
+        class="main_content about_us"
+        id="about_us"
+      >
+        <div class="main_init_info">
+          <h1 class="header_section_base header_section_first">
+            Move Free
+          </h1>
```

Comment:
This tag, as a rule, is used on the page only 1 time and in one place, because above you already use it, so in all other places you need to use a different level heading
```suggestion
          <h2 class="header_section_base header_section_first">
            Move Free
          </h2>
```